### PR TITLE
Avatar and names added to contact page

### LINKED ISSUE #172: Show contact's name and picture on the contact page
Might make the contact page a little bit säxier by adding the name and picture of that contact.

### PR DESCRIPTION
**Pull request by davidkrantz**
Updates the contact page with an avatar and names of the persons who have the post, see picture below. For this to work [PR](https://github.com/fsek/web/pull/915) is needed. Also, closes #172 

skarmavbild 2019-02-13 kl 17 53 05

### CODE REVIEW

File: www/scss/partials/_contact.scss [hunk 1]
```diff
@@ -32,13 +37,42 @@
...
+  background-repeat: no-repeat;
+  background-image: url("../img/missing_thumb.png");
+  margin: auto;
+  box-shadow: 0px 2px 10px 2px rgba(0,0,0,0.2);
+  border-radius: 100%;
+  border: 3px solid #d6d6d6;
+}
+
+.contact-container {
+  width: 100%;
```
Comment (FredrikLastow) [inline www/scss/partials/_contact.scss]: You don't need to define the width here I don't think.
```suggestion

```

File: www/scss/partials/_contact.scss [hunk 2]
```diff
@@ -32,13 +37,42 @@
 .contact-content .block-title {
   color: $fsek-orange;
   font-size: 17px;
+  overflow: visible;
 }
 
 .contact-content .contact-button {
   background-color: $fsek-orange !important;
   margin-top: 20px;
```
Comment (FredrikLastow) [inline www/scss/partials/_contact.scss]: Add a `margin-bottom` here for material so the button isn't on the complete bottom of the screen

File: www/scss/partials/_contact.scss [hunk 3]
```diff
@@ -32,13 +37,42 @@
 .contact-content .block-title {
   color: $fsek-orange;
   font-size: 17px;
+  overflow: visible;
```
Comment (FredrikLastow) [inline www/scss/partials/_contact.scss]: Is this needed?

Comment (davidkrantz) [reply www/scss/partials/_contact.scss]: Yes, it seems like it.

**Pull request merged**